uber/models.py

```diff
@@ -771,7 +771,6 @@ class Attendee(MagModel, TakesPaymentMixin):
     region        = Column(UnicodeText)
     country       = Column(UnicodeText)
     no_cellphone  = Column(Boolean, default=False)
```

Comment:
I have plans!  There is now an API for event plugins to add their own db columns, just like MAGStock is doing now.  They can also add their own config, add costs to attendees and groups, add or replace validations, etc.  The idea is that there's an API for all of this so that we don't have to jam everything into the core Uber plugin.

As soon as we launch MAGStock prereg and we have some breathing room I'll explain my approach and we can go back and forth to adjust/tweak it as necessary so it makes sense for everyone and fits everyone needs and use cases.